### PR TITLE
Fix auto starting arm64 ubuntu jobs

### PR DESCRIPTION
**Pull request by jashook**

### CODE REVIEW

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test CI Please

File: netci.groovy [hunk 1]
```diff
@@ -1204,8 +1204,12 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                 case 'Windows_NT':
                     switch (scenario) {
                         case 'default':
+                            if (isFlowJob == true) {
+                                Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} ${configuration} Priority 1 Build and Test", "(?i).*test\\W+${os}\\W+${scenario}.*")
```
Comment (RussKeldorph) [inline netci.groovy]: The trigger string doesn't seem like it ought to be hard coded like this.

**Comment by jashook** [comment]
@RussKeldorph addressed ptal when you can.

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Pull request merged**